third_party/types/common/errors.go

```diff
@@ -29,4 +29,5 @@ var (
 	ErrListenerRequired          = errors.New("listener was not Provided")
 	ErrMDNSInvalidConn           = errors.New("invalid Connection, cannot begin MDNS Service")
 	ErrMotorWalletNotInitialized = errors.New("motor Wallet is not initialized")
+	ErrDefaultStillImplemented   = errors.New("default implementation of event callback still defined")
```

Comment:
I think the change in this message makes it less clear that there is a problem. If the default implementation is being used, I think it's fair to error with "callback not implemented"